src/fbw/src/FlyByWireInterface.cpp

```diff
@@ -1255,6 +1257,16 @@ bool FlyByWireInterface::updateAutothrust(double sampleTime) {
   idAutothrustMode->set(autoThrustOutput.mode);
   idAutothrustModeMessage->set(autoThrustOutput.mode_message);
 
+  // update warnings
+  auto fwcFlightPhase = idFwcFlightPhase->get();
+  if (fwcFlightPhase == 2 || fwcFlightPhase == 3 || fwcFlightPhase == 9) {
```

Comment:
While you're right that these are the inhibited flight phases, inhibition is not the same as activation. If this message was active in a previous flight phase it can in fact stay active in some subsequent flight phases (inhibitions are only for new messages). The correct flight phase activation is as follows:

```suggestion
  if (fwcFlightPhase == 2 || fwcFlightPhase == 3 || fwcFlightPhase == 4 || fwcFlightPhase == 8 || fwcFlightPhase == 9) {
```